src/main/java/org/mule/tools/devkit/sonar/checks/java/MinMuleVersionCheck.java

```diff
@@ -0,0 +1,38 @@
+package org.mule.tools.devkit.sonar.checks.java;
+
+import com.google.common.collect.Iterables;
+import com.google.common.collect.Lists;
+import org.mule.tools.devkit.sonar.utils.ClassParserUtils;
+import org.sonar.check.Priority;
+import org.sonar.check.Rule;
+import org.sonar.java.model.expression.AssignmentExpressionTreeImpl;
+import org.sonar.java.model.expression.IdentifierTreeImpl;
+import org.sonar.plugins.java.api.tree.*;
+import org.sonar.squidbridge.annotations.ActivatedByDefault;
+
+import java.util.List;
+
+@Rule(key = MinMuleVersionCheck.KEY, name = "Connectors should specify a minMuleVersion", description = "The minMuleVersion should be specified on the @Connector annotation.", priority = Priority.CRITICAL, tags = { "connector-certification" })
```

Comment:
Minor fixes for the description: The **'minMuleVersion'** should be specified **in** the @Connector annotation.